src/Database/Database.php

```diff
@@ -825,6 +825,7 @@ public function e(string $sql): bool
 	 *
 	 * @return boolean Are there rows for that condition?
 	 * @throws \Exception
+	 * @todo Get rid of the DBStructure::existsTable() invocation
```

Comment:
Can you please expand on the why or provide a contact information should anyone require more information from you?

Reply:
This invocation is redundant and is just wrapped by `DBA::exists()`.

Reply:
Thank you, I meant in the code itself. 😄

Reply:
For me, I differentiate between no table created and no records found in an existing table. `DBA::exists()` makes both cases the same. We should explicitly test if no table is there (incomplete installation/update) or if no record in an existing table was found.

Reply:
Thank you for the further elaboration, I just meant that you should ass the following to the `@todo` comment you added in this PR:

> This invocation is redundant and is just wrapped by `DBA::exists()`.